### ISSUE TITLE
Stress Test for Video Playing

### ISSUE THREAD

**Issue by trchiao** [the-core-studio/vox-issue#206]
Conduct stress tests to ensure video play function for large amount of users.

**Comment by tclzcja** [comment]
@trchiao 

I've spoken with Mike on this. Technically speaking, the stress for video playing does not exists on our Vox system, instead, it's on TALIC's cloud server, which I don't have any way to optimise or control.

Plus, the infrastructure of HKU - WiFi, Router - also play a role in the video streaming performance. Meaning if there're 100 people playing a video in the same classroom, I'm not sure whether the WiFi router they access to has enough capacity to stream 100 video at the same time.

In conclusion, Vox server and system has almost nothing to do with the performance of video playing. Vox simply gets a URL from the database and show it to user, that's pretty much it. I don't think there's anything I can optimise.

**Comment by tclzcja** [comment]
Closing this one.
